### PR TITLE
Shruti assignment20- 90 minutes

### PR DESCRIPTION
**Pull request by ShrutiDJun20**
Output of ArmstrongNumber:
image

Output of MissingNumberinArr:
image

Output of ReverseOfInteger:
image

Output of findDuplicateNumber:

image

### CODE REVIEW

File: src/shrutiDJun20Assignments/ReverseOfInteger.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	
+	void findReverse(int input){
+		int number2 = input;
+		int revnum = 0;
+		while(number2>0){  //235
+			//number=number2%10;   //235%10=5
+			//number2=number2/10;
+			
+			revnum = (revnum*10)+(number2%10);
+			number2 = number2/10;
```
Comment (PalakSJun20) [inline src/shrutiDJun20Assignments/ReverseOfInteger.java:13]: You can write proper message what is the input and what is the output of that.

**Review by PalakSJun20 — APPROVED** [review]
Approved

File: src/shrutiDJun20Assignments/ReverseOfInteger.java (hunk 1 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20Assignments/ReverseOfInteger.java:13]: changes done as suggested

File: src/shrutiDJun20Assignments/ArmstrongNumber.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+public class ArmstrongNumber {
+	
+	void isNumberArmstrong(int input){
+	//number that is equal to the sum of cubes of its digits
+		//1^3 + 5^3 + 3^3 = 153
+		int sum = 0;
+		int num=0;
+		int num2=input;
+		while(num2>0){
+			num= (num2%10);     //153%10=3(num)   //5=15%10    //1=1%10
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20Assignments/ArmstrongNumber.java:12]: here "(" is not required.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me. Keep it up.

**Pull request merged**